src/_span.scss

```diff
@@ -0,0 +1,13 @@
+
+@mixin vui-gs-span($span: 1) {
+
+	@include span($span);
+
+	$temp-span: parse-span($span);
```

Comment:
Here, the $span (which could be short-hand) is parsed into a map.  The map is needed because we need to merge the flow (RTL) setting immediately below (because it is not possible to merge short-hand directly with a value or another map.